### PR TITLE
ci(postman): Add save card flow testcases

### PR DESCRIPTION
**Pull request by Gnanasundari24**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [x] CI/CD

## Description
Added save card flow testcases for stripe,adyen,checkout,bluesnap connector

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## How did you test it?
Ran the collections
**Stripe**
Screenshot 2023-10-12 at 5 14 05 PM

**Adyen**
Screenshot 2023-10-12 at 5 29 12 PM

**Bluesnap**
Screenshot 2023-10-12 at 5 38 33 PM

**Checkout**
Screenshot 2023-10-12 at 5 41 42 PM

## Checklist

- [ ] I formatted the code `cargo +nightly fmt --all`
- [ ] I addressed lints thrown by `cargo clippy`
- [ ] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: adyen_uk/.auth.json
Comment (pixincreate) [inline adyen_uk/.auth.json:1]: Looks like you exported Adyen collection and and never moved to `postman/collection-dir`.
Kindly remove this folder.

File: postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario10-Partial refund/Payments - Create/.event.meta.json
Comment (pixincreate) [inline postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario10-Partial refund/Payments - Create/.event.meta.json:1]: Rename `Scenario10-Partial refund` `Scenario11-Partial refund`

File: postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario20-Pass Invalid CVV for save card flow and verify failed payment/.meta.json
Comment (pixincreate) [inline postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario20-Pass Invalid CVV for save card flow and verify failed payment/.meta.json:1]: Can this (`Scenario20` and `Scenario21`) be moved to Variation testcases? It looks like it

**Review by pixincreate — CHANGES_REQUESTED** [review]

File: postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario20-Pass Invalid CVV for save card flow and verify failed payment/.meta.json
Comment (Gnanasundari24) [reply postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario20-Pass Invalid CVV for save card flow and verify failed payment/.meta.json:1]: No its not a 4xx testcases. It is happy cases with failure payments

File: postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario11-Partial refund/Refunds - Create-copy/request.json
Comment (pixincreate) [inline postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario11-Partial refund/Refunds - Create-copy/request.json:1]: Can this kind of changes be reverted?

File: postman/collection-dir/checkout/Flow Testcases/Happy Cases/Scenario11-Save card flow/Payments - Retrieve-copy/event.test.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    function () {
+      pm.expect(jsonData.amount).to.eql(6540);
+    },
+  );
+}
+
+// Response body should have value "6000" for "amount_received"
+if (jsonData?.amount_received) {
+  pm.test(
+    "[POST]::/payments:id/capture - Content check if value for 'amount_received' matches '6000'",
```
Comment (pixincreate) [inline postman/collection-dir/checkout/Flow Testcases/Happy Cases/Scenario11-Save card flow/Payments - Retrieve-copy/event.test.js:91]:
```suggestion
    "[POST]::/payments:id/capture - Content check if value for 'amount_received' matches '6540'",
```

File: postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario19-Add card flow/Payments - Retrieve Copy/event.test.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    function () {
+      pm.expect(jsonData.amount).to.eql(6540);
+    },
+  );
+}
+
+// Response body should have value "6000" for "amount_received"
+if (jsonData?.amount_received) {
+  pm.test(
+    "[POST]::/payments:id/capture - Content check if value for 'amount_received' matches '6000'",
```
Comment (pixincreate) [inline postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario19-Add card flow/Payments - Retrieve Copy/event.test.js:91]:
```suggestion
    "[POST]::/payments:id/capture - Content check if value for 'amount_received' matches '6540'",
```

File: postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario20-Pass Invalid CVV for save card flow and verify failed payment/Payments - Retrieve/event.test.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    function () {
+      pm.expect(jsonData.amount).to.eql(6540);
+    },
+  );
+}
+
+// Response body should have value "6000" for "amount_received"
+if (jsonData?.amount_received) {
+  pm.test(
+    "[POST]::/payments:id/capture - Content check if value for 'amount_received' matches '6000'",
```
Comment (pixincreate) [inline postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario20-Pass Invalid CVV for save card flow and verify failed payment/Payments - Retrieve/event.test.js:91]:
```suggestion
    "[POST]::/payments:id/capture - Content check if value for 'amount_received' matches '6540'",
```

**Review by pixincreate — COMMENTED** [review]
Other than that, LGTM.

File: postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario20-Pass Invalid CVV for save card flow and verify failed payment/Payments - Retrieve/event.test.js (hunk 3 above)
Comment (Gnanasundari24) [reply postman/collection-dir/adyen_uk/Flow Testcases/Happy Cases/Scenario20-Pass Invalid CVV for save card flow and verify failed payment/Payments - Retrieve/event.test.js:91]: This is a test description. It won't cause a test failure, as it only involves running a function with the correct input value, ensuring the expected output is produced.

**Review by SanchithHegde — APPROVED** [review]

**Review by pixincreate — APPROVED** [review]

**Review by SanchithHegde — APPROVED** [review]

**Pull request merged**